Challenge-4/Bot_2/src/agent.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  HandleTransaction,
+  TransactionEvent,
+  ethers,
+  Finding,
+  getEthersProvider,
+} from "forta-agent";
+import {
+  BORROW_RATE,
+  SUPPLY,
+  SUPPLY_RATE,
+  USDC_TOKEN_ETH,
+  UTILIZATION,
+  WITHDRAW,
+} from "./constants";
+import Helper from "./helper";
+import CONFIG from "./agent.config";
+import { NetworkData } from "./types";
+import { NetworkManager } from "forta-agent-tools";
+import {
+  supplyFinding,
+  borrowFinding,
+  alertSupplyFinding,
+  alertBorrowFinding,
+} from "./findings";
+
+let configuratorProxy: string | undefined;
+let tokenAddress: string;
+let helper: Helper;
+const networkManager = new NetworkManager<NetworkData>(CONFIG);
```

Comment:
This could be assigned the value in the `initialize()`, so in the tests you can use mock values, instead of having to use the actual USDC address, for example. (Would need to be passed to `provideInitialize()` as an argument)